src/index.ts

```diff
@@ -38,6 +38,17 @@ async function run(): Promise<void> {
     const token = getInput('token', { required: true });
     const client = new GitHub(token);
     const pullRequest = context.issue;
+    const { sender } = context.payload;
+
+    // Exempt Users
+    const exemptUsers = getInput('exemptUsers', { required: true }).split(',');
```

Comment:
Split this and then trim the parts since people are likely to use a comma followed by a space.